README.md

```diff
@@ -64,14 +64,29 @@ index-import \
 
 ## Build Instructions
 
-The build uses [CMake](https://cmake.org) because [swift-clang](https://www.github.com/apple/swift-clang) uses CMake. The `index-import` build script is small, but depends on the larger swift-clang project. To build `index-import`, first build swift-clang using the [Swift build instructions](https://github.com/apple/swift#system-requirements). Be sure to checkout a release branch of swift, not `master`, for example `swift-5.0-branch`.
+The build uses [CMake](https://cmake.org) because [swift-clang](https://www.github.com/apple/swift-clang) uses CMake. The `index-import` build script is small, but depends on the larger swift-clang project. To build `index-import`, first [install the tools required by Swift](https://github.com/apple/swift#system-requirements), then build swift-clang by following the [Swift build instructions](https://github.com/apple/swift#building-swift).
 
-Once swift has been built, `index-import` can be built as follows. The key step is to add the llvm `bin` directory from the swift build, to your `PATH`. This ensures CMake can find the build files it needs.
+When building Swift, keep the following in mind:
 
+1. Checkout the desired release branch of Swift
+2. Build Swift using `--release`/`-R` for performance
+
+To checkout a specific version of Swift, use the `update-checkout` script. For example: `./swift/utils/update-checkout --scheme swift-5.0-branch`.
```

Comment:
Isn't it a bit safer to do `--clone --tag swift-5.0.1-RELEASE` since the branch can be ahead of the tagged release? Not a biggie but that's what I do.

Reply:
Good point, I'll change it.

Reply:
fixed, thanks for catching that